libs/core/src/lib/select/select.component.html

```diff
@@ -12,9 +12,13 @@
             <button class="fd-select-button-custom"
                     fd-button
                     aria-haspopup="true"
-                    [ngClass]="{'fd-button--compact': compact}"
                     [attr.aria-expanded]="isOpen"
-                    [disabled]="disabled">
+                    [disabled]="disabled"
+                    [ngClass]="{'fd-button--compact': compact}"
```

Comment:
We should be consistent here. when there was one class added using `ngClass` We should use the same semantics to add other classes. There is no reason to use `class.<attribute >` syntax

So what you want is:

```html
[ngClass]=“{
  ‘fd-button—compact’: compact,
  ‘<YOU_CLASS>1’: ‘<YOUR CONDITION1>’,
  ‘<YOU_CLASS>‘: ‘<YOUR CONDITION2>’

}
”
```